aptos-move/framework/aptos-framework/sources/vesting.move

```diff
@@ -417,7 +417,15 @@ module aptos_framework::vesting {
 
         let vesting_contract = borrow_global_mut<VestingContract>(contract_address);
         let contract_signer = &get_vesting_account_signer_internal(vesting_contract);
-        staking_contract::unlock_rewards(contract_signer, vesting_contract.staking.operator);
+        // Request commission first to make it easier to calculate what's the accumulated rewards that belong to the
+        // vesting pool.
+        let operator = vesting_contract.staking.operator;
+        staking_contract::request_commission(contract_signer, contract_address, operator);
```

Comment:
do we need this since unlock_stake already does a request_commission internally? isn't the accumulated_rewards = total_active_stake - remaining_grant - commission_amount?

Reply:
Great suggestion, thanks! Definitely a simpler and less redundant computation. Updated.